src/views/store/Index.vue

```diff
@@ -1,81 +1,144 @@
 <template>
-    <Searchbar 
-    :categories="categories" 
-    :categorySelected="categorySelected"
-    :onChangeCategory="onChangeCategory"
-    :onChangeName="filterByName"
-    :search="search"
-    :height=" windowWidth < 1024 ? '48' : '80'"
-    />
+  <div class="index">
+    <Header />
+    <div class="indexMainSection">
+      <Searchbar
+        :categories="categories"
+        :categorySelected="categorySelected"
+        :onChangeCategory="onChangeCategory"
+        :onChangeName="filterByName"
+        :search="search"
+        :height="windowWidth < 1024 ? '48' : '80'"
+      />
+      <div v-if="categorySelected == 'TODOS'" class="products">
+        <Product
+          v-for="i in 3"
+          :key="i"
+          :category="categories[i]"
+          :products="products.filter((product) => product.category === categories[i])"
+        />
+      </div>
+      <Product
+        v-else
+        :category="categorySelected"
+        :products="products.filter((product) => product.category === categorySelected)"
+      />
+    </div>
+    <Footer />
+  </div>
 </template>
 
 <style lang="scss" scoped>
-    .search {
-        height: 100px;
-    }
+.search {
+  height: 100px;
+}
+.indexMainSection {
+  padding-top: 10rem;
+}
 </style>
 
 <script>
-import Searchbar from '../../components/store/Searchbar'
+import Searchbar from "../../components/store/Searchbar";
+import Header from "../../components/Header";
+import Product from "../../components/store/Products";
+import Footer from "../../components/Footer";
+
 export default {
   components: {
     Searchbar,
+    Header,
```

Comment:
I think we should load these components in the store view instead of in each component.